### ISSUE TITLE
The "BEAT THE RUSH" achievement isn't working.

### ISSUE THREAD

**Issue by LionDenChristian** [ValveSoftware/Source-1-Games#5146]
I've done the achievement requirements to my knowledge, and it never works. It seems everyone online has had the same issue for years now. I'd like to be able to get the achievement in the game, but at some point in some update or something, it seemed to break for everyone. The only way I know to get it is through mods and glitches. Can someone fix this.

**Comment by NULLYUKI** [comment]
It would be nice if you could clarify for which game the achievement is.

**Comment by LionDenChristian** [comment]
> It would be nice if you could clarify for which game the achievement is.

Sorry I never post so I didn't know, but a mod label it already thankfully.

**Comment by Tsuey** [comment]
Some internal documentation we have about this Achievement, investigated by @Orinuse -- in short, it is erroneously voided in several ways, and is possible but only with narrow precision, hence changing the criteria for the Achievement https://github.com/Tsuey/L4D2-Community-Update/issues/387 would be more realistic than fixing what's there:

> BEAT THE RUSH
> In a Survival round, get a medal only using melee weapons.
> Does not unlock correctly.
> 
> 1.The achievement is strictly tied to unlocking a medal, rather than reaching the time of a medal. If a player already has silver medals on every map, they would have to play for 10 minutes to unlock it with the gold medal
> 
> 2.The following fails the achievement:
> - In some maps, the moment you spawn in the achievement automatically fails, however this does not happen when loading into some maps: 1."c8m1_apartments" (coop/VS mode), 2. "c8m2_subway", 3. "c11m5_runway". A theory as to why this happens is when spawning, you as a survivor do not spawn on nav areas marked as BATTLESTATION, FINALE or CHECKPOINT. (Why?)
> - At any point, any damage a survivor  takes or inflict voids the achievement - Shooting a friend with a pistol voids, taking damage from Tank's punches voids. Exceptions to this are shoving or stomping common infected to death. (Why?)
> - The achievement has already been failed once in the session, and the map hasn't been changed yet. (Why?) 
> - After a round has started, anyone uses "+attack" on a non-melee weapon.
> - Dying. 
> 
> Additionally, these were tested to see if they are treated as achievement criterias, but ended up not being one:
> - Using only melee weapons that naturally spawned in the map (i.e. not using melees given with admin commands or placed via a lump file)
> - Not picking up anything except a melee
> - Playing specifically on a survival map from before TLS
> - Avoiding the Knife, Pitchfork and Shovel
> - Playing on a mode that isn't survival.
> 
> Screenshot showing the achievement is still technically achievable: https://steamcommunity.com/sharedfiles/filedetails/?id=2806079445
> Failzz tested this achievement in an older version of L4D2 and the achievement works completely normally without issue.
> Needs heavy testing.
>

**Comment by Tiagoquix** [comment]
Hi OP, is this still relevant as of 2024?

**Comment by LionDenChristian** [comment]
> Hi OP, is this still relevant as of 2024?

Yes, I cant personally test it, but I looked into it and I see people talking about it in the past few months.
